frontend/src/metabase/lib/time.js

```diff
@@ -123,3 +155,15 @@ export function formatFrame(frame) {
 export function getRelativeTime(timestamp) {
   return moment(timestamp).fromNow();
 }
+
+export function getRelativeTimeAbbreviated(timestamp) {
+  const locale = moment().locale();
```

Comment:
This code is really annoying to write unit tests for because of the way `moment` handles `locale` globally. If anyone would like, I can do something hacky like `export const _moment_from_module_ = moment` so that the unit test has access to the same version of `moment` as the module. Otherwise, we can wait for the Jest update to go through and mocking will become easier with `jest.requireActual`